src/app/libp2p_helper/src/codanet.go

```diff
@@ -171,6 +184,12 @@ func (gs *CodaGatingState) InterceptPeerDial(p peer.ID) (allow bool) {
 // resolved the peer's addrs, and prior to dialling each.
 func (gs *CodaGatingState) InterceptAddrDial(id peer.ID, addr ma.Multiaddr) (allow bool) {
 	allow = (!gs.DeniedPeers.Contains(id) || gs.AllowedPeers.Contains(id)) && !gs.AddrFilters.AddrBlocked(addr)
+
+	if !allow {
+		gs.logger.Infof("disallowing peer dial from: %v", p)
```

Comment:
Isn't the equivalent of this logged [here](https://github.com/libp2p/go-libp2p-swarm/blob/830b5b61da7fee5e6ead8b840399f1895e1aa3d3/swarm_dial.go#L228) already?

Reply:
These are different layers the connection gating goes through. If we reject the first one, we won't get here.